### DISCUSSION TITLE
MIDI

### DISCUSSION THREAD

**Discussion by ScottHsuMusic** [SynthstromAudible/DelugeFirmware#2709] [Ideas]
Synthstrom discontinued their Deluge footswitch a while ago, and it's been pretty hard to find a road worth MIDI controller for the Deluge. Currently the MIDI global midi commands on the deluge don't respond to CC messages, which is what the majority of MIDI footswitches use, but rather note on/off messages. I've been using a mini USB MIDI keyboard as a footswitch with notes on the keyboard mapped to the LOOP and LAYER commands. This works ok at home but I'd rather use a footswitch in a metal enclosure so it doesn't break.

Wondering if this could be fixed on the firmware side? Change the way MIDI commands work to accept CC?

**Comment by seangoodvibes** [comment]
You can definitely learn cc's to global midi commands in the community firmware. I just looked in the firmware code and learning cc to global midi commands is definitely a thing. 

Are you coming from the official firmware?

**Reply by ScottHsuMusic** [reply]
Nope I've been on the community firmware for a while now, never noticed this but I see it now in the feature list. Will test it out this week!
